firedrake/mg/kernels.py

```diff
@@ -29,7 +30,7 @@
 from finat.quadrature import make_quadrature
 from firedrake.pointquery_utils import dX_norm_square, X_isub_dX, init_X, inside_check, is_affine, compute_celldist
 from firedrake.pointquery_utils import to_reference_coordinates as to_reference_coordinates_body
-from firedrake.utils import ScalarType_c, complex_mode
+from firedrake.utils import ScalarType_c, ScalarType
```

Comment:
Why do we have 3 import lines for `firedrake.utils` (l.5, 8 and 33), can we not just write down everything in a single line?

Reply:
Sorry. I edited with sed and forgot to check if I did something stupid. Fixed now.